src/nyc_trees/apps/quiz/models.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+
+class Quiz(object):
```

Comment:
I put `AchievementDefinition` in `users/models.py` because they very directly relate to the `Achievement` Django Model.

Since that does not apply here, that might not be the right idea.  Should this live in the same file as `QuizResult`?